src/FBAppEventsLogger.ts

```diff
@@ -131,7 +133,7 @@ module.exports = {
   logEvent(eventName: string, ...args: Array<number | Params>) {
     let valueToSum = 0;
     if (typeof args[0] === 'number') {
-      valueToSum = args.shift();
+      valueToSum = args.shift() as number;
```

Comment:
Maybe instead of using ’as’ here, you could use ’Number(args.shift()’
What do you think?